pallets/token-manager/src/migration.rs

```diff
@@ -0,0 +1,70 @@
+use crate::{Config, Pallet, LowerSchedulePeriod};
+use frame_support::{
+    dispatch::GetStorageVersion,
+    pallet_prelude::{PhantomData, StorageVersion},
+    traits::{Get, OnRuntimeUpgrade},
+    weights::Weight,
+};
+
+pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
+
+
+pub fn set_lower_schedule_period<T: Config>() -> Weight {
+    let default_lower_schedule_period: T::BlockNumber = 3600u32.into(); // ~ 12 hrs
```

Comment:
```suggestion
    let default_lower_schedule_period: T::BlockNumber = 3275u32.into(); // ~ 12 hrs
```

12 seconds is theoretical, real block production time is ~13.19 seconds on mainnet presently